frontend/src/Layout.tsx

```diff
@@ -172,6 +180,18 @@ function Layout(props: LayoutProps): JSX.Element {
     );
   }
 
+  function RenderSearchbar(): JSX.Element | null {
+    if (currentPath === '/home') {
```

Comment:
Why is there no search on the startpage?

Reply:
Based on the user story #30 the search field should be displayed on every subpage. In my opinion, this affects all pages after the app entry page - Home.

Reply:
No, this just states that there should not be a dedicated search page, but the search shall be accessible from every page.